code/modules/vending/_vending.dm

```diff
@@ -1148,11 +1151,6 @@ GLOBAL_LIST_EMPTY(vending_products)
 			last_slogan = world.time + rand(0, slogan_delay)
 			return
 
-		if(canLoadItem(I))
-			loadingAttempt(I,user)
-			updateUsrDialog()
-			return
-
```

Comment:
What's uh, what's this?

Reply:
oh my god I see

Reply:
if you want me to do that in another pr thats fine. but because of this being here vending/custom/canLoadItem() is called twice, once in vending/custom/attackby and vending/attackby. so if the proc returns false for any reason it will say the failing line twice in a row.

Reply:
Ok uh, A: Are you sure this isn't changing behavior outside of the normal? It looks like the loading proc on the base has some if checks around it that this doesn't have.
B: Why the hell is the same check for wires and a wire tool being done both here and at the very start of the standard attack_by?

Reply:
A: i cant find any negative behavior this could cause either via testing or reading it, in this version there are two checks before the canLoadItem() call, the first is for the wire check that you pointed out, which doesnt matter because we dont want to check if that can be loaded if the panel is open, and the other is the check for if its a refill canister, which should be better since we dont want to load the refill canister itself into the vending machine (canLoadItem does not return false for the vending machines refill_canister)

B: shitcode that i missed at first, taken out now